src/Installers/Windows/AspNetCoreModule-Setup/Directory.Build.props

```diff
@@ -70,5 +70,20 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <SetPlatform>Platform=Win32</SetPlatform>
     </ProjectReference>
+   
+    <ProjectReference Include="$(_ServerIISBasePath)AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj">
```

Comment:
nit: This file is much more verbose than it needs to be. Could we please switch to using attributes on the `<ProjectReference/>` elements soon❔

Reply:
If you tell me what you want to do here, I'm happy to clean that up the next time I do an installers PR

Reply:
e.g. for the last `<ProjectReference />` element&hellip;

``` msbuild
    <ProjectReference Include="$(_ServerIISBasePath)AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj"
        Name="AspNetCoreV2HandlerARM64"
        Private="True"
        DoNotHarvest="True"
        ReferenceOutputAssembly="false"
        SetPlatform="Platform=ARM64" />
```